src/notebook_unit1.ipynb

```diff
@@ -95,9 +95,9 @@
    "source": [
```

Comment:
<p>Normally <code>nstate_view = np_nstate.view(dtype=np.complex128)</code> should be enough. Here it doesn't work due to the pandas dataframe giving a Fortran-ordered numpy array instead of a C-ordered one. Maybe use <code>np.nstate = np.ascontiguousarray(nstate.values)</code> to avoid the Fortran style ordering or don't use pandas for nstate at all. Use e.g. <code>np.loadtxt</code> (uses whitespace as delimiter per default) to directly create an array </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pauleonix/ssd_course/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>